docs/source/user-guide/tasks/manage-virtual.rst

```diff
@@ -14,6 +14,7 @@ small bit of code to detect the presence or absence of the system feature that
 corresponds to the package.  The currently supported list of virtual packages includes:
 
   * ``__cuda``: Maximum version of CUDA supported by the display driver
+  * ``__glibc``: Version of globc supported by the OS
```

Comment:
Shouldn't it just be called `__libc`? Because someday it might be e.g. `musl-libc` as `libc_family`.

Reply:
Oh, I see how this is implemented. Nevermind. Sorry for the noise.